src/transformers/trainer_utils.py

```diff
@@ -53,3 +55,39 @@ class TrainOutput(NamedTuple):
 
 
 PREFIX_CHECKPOINT_DIR = "checkpoint"
+
+
+class FinalActivation(ExplicitEnum):
+    """
+    Possible values for the ``final_activation`` argument in :meth:`Trainer.from_nlp_dataset`.
+    Useful for tab-completion in an IDE.
+    """
+
+    NONE = "none"
+    ARGMAX = "argmax"
+    SIGMOID = "sigmoid"
+    SOFTMAX = "softmax"
```

Comment:
Thought it was better to have some kind of enum for possible final activation functions than letting the user provide their own.